truss/templates/shared/util.py

```diff
@@ -85,3 +89,16 @@ def kill_child_processes(parent_pid: int):
 
 def transform_keys(d: Dict[X, Z], fn: Callable[[X], Y]) -> Dict[Y, Z]:
     return {fn(key): value for key, value in d.items()}
+
+
+def download_from_url_using_requests(URL: str, download_to: Path):
```

Comment:
why do we need this duplicated? is there some weird module import scenario?

Reply:
We need to duplicate this utility because of how our templating story is set up – this `shared` subdir doesn't have truss as a dependency and when we use it in the actual container environments we generate doesn't actually know about the truss lib and it's utils